### ISSUE TITLE
After migrating to fly.io, flyctl only works partially

### ISSUE THREAD

**Issue by robster7674** [nightscout/cgm-remote-monitor#7759]
**If you need support for Nightscout, PLEASE DO NOT FILE A TICKET HERE**
For support, please post a question to the "CGM in The Cloud" group in Facebook
(https://www.facebook.com/groups/cgminthecloud) or visit the WeAreNotWaiting Discord at https://discord.gg/zg7CvCQ

**Describe the bug**

After following https://nightscout.github.io/vendors/fly.io/migrate/, NightScout runs fine, but when I want to work with the app through flyctl, most commands don't work. Case in point: I want to change a config var, so I would need to work with the secrets. However, when I type "fly secrets list", I get:
```
% fly secrets list
Error Could not find App
```

**To Reproduce**
1. Follow https://nightscout.github.io/vendors/fly.io/migrate/
2. Install flyctl using brew
3. Login using "fly auth login"
4. Type "fly secrets list"

**Expected behavior**
Be able to list secrets.

**Screenshots**
n/a

**Your setup information**
n/a

**Additional context**
In short: how do I get flyctl to work with the migrated Heroku app?

**Comment by robster7674** [comment]
Looks like a user error... :) Using https://nightscout.github.io/vendors/fly.io/new_user/#secrets I can now list the secrets:
```
 % fly secrets list -a appname
NAME                     	DIGEST          	CREATED AT           
ALARM_HIGH               	20fc1a8f6e9b1c21	2022-12-03T12:35:04Z	
ALARM_LOW                	20fc1a8f6e9b1c21	2022-12-03T12:35:03Z	
ALARM_TIMEAGO_URGENT     	20fc1a8f6e9b1c21	2022-12-03T12:35:04Z	
ALARM_TIMEAGO_URGENT_MINS	b134196ffeb28941	2022-12-03T12:35:03Z	
ALARM_TIMEAGO_WARN       	20fc1a8f6e9b1c21	2022-12-03T12:35:04Z	
ALARM_TIMEAGO_WARN_MINS  	5fdae8f42a46935a	2022-12-03T12:35:02Z	
ALARM_TYPES              	8e296e5e0353eb22	2022-12-03T12:35:03Z	
ALARM_URGENT_HIGH        	20fc1a8f6e9b1c21	2022-12-03T12:35:01Z
...
```
Now figuring out how to use secrets set.
